Style/10 gráfica votación

NoSoyDani opened the pull request.

NoSoyDani committed:
feat: Añade ficheros base para representar gráficas

Se ha añadido los ficheros necesarios para implementar la funcionalidad.

Se ha añadido la funcionalidad básica a dichos ficheros.

NoSoyDani committed:
feat: Añade la representación textual de los valores

Se ha añadido la funcionalidad necesaria para mostrar los
valores que debe de representar la gráfica.

NoSoyDani committed:
feat: Añadida plantilla de gráfica

Se ha añadido la funcionalidad necesaria para mostrar una gráfica, por ahora,
dicha gráfica posee únicamente datos de prueba.

eveyugyug commented on decide_panel/src/components/Bar.js:
```diff
@@ -0,0 +1,55 @@
+                    
+                ],
+                borderColor: [
+                    'rgba(255, 99, 132, 1)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(255, 99, 132, 1)",
```

eveyugyug replied:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(255, 99, 132, 1)",
```

eveyugyug commented on decide_panel/src/components/Bar.js:
```diff
@@ -0,0 +1,55 @@
+                label: "Votos",
+                data: [1,1],
+                backgroundColor: [
+                    'rgba(255, 99, 132, 0.2)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(255, 99, 132, 0.2)",
```

eveyugyug replied:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(255, 99, 132, 0.2)",
```

eveyugyug commented on decide_panel/src/components/Bar.js:
```diff
@@ -0,0 +1,55 @@
+import Api from "../services/backend";
+import React from "react";
+import { Bar } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
import Chart from "chart.js/auto";
```

eveyugyug replied:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
import Chart from "chart.js/auto";
```

eveyugyug commented on decide_panel/src/components/Bar.js:
```diff
@@ -0,0 +1,55 @@
+                ],
+                borderColor: [
+                    'rgba(255, 99, 132, 1)',
+                    'rgba(54, 162, 235, 1)'
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(54, 162, 235, 1)"
```

eveyugyug replied:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(54, 162, 235, 1)"
```

eveyugyug commented on decide_panel/src/components/Bar.js:
```diff
@@ -0,0 +1,55 @@
+                data: [1,1],
+                backgroundColor: [
+                    'rgba(255, 99, 132, 0.2)',
+                    'rgba(54, 162, 235, 0.2)'
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(54, 162, 235, 0.2)"
```

eveyugyug replied:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                    "rgba(54, 162, 235, 0.2)"
```

NoSoyDani committed:
feat: Añade funcionalidad necesaria para mostrar la relación de votos en una gráfica

Se ha desarrollado la funcionalidad necesaria para mostrar la relación de los
votos en un gráfico de tipo tarta.

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                  label: "",
+                  data: tally,
+                  backgroundColor: [
+                      'rgba(255, 99, 132, 0.2)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(255, 99, 132, 0.2)",
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                      'rgba(255, 99, 132, 1)',
+                      'rgba(255, 99, 132, 1)',
+                      'rgba(255, 99, 132, 1)',
+                      'rgba(255, 99, 132, 1)'
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(255, 99, 132, 1)"
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                      'rgba(50, 250, 250, 0.2)',
+                      'rgba(23, 99, 250, 0.2)',
+                      'rgba(250, 99, 250, 0.2)',
+                      'rgba(250, 0, 0, 0.2)'
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(250, 0, 0, 0.2)"
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+import { ProgressBar } from "primereact/progressbar";
+import React from "react";
+import { Pie } from "react-chartjs-2";
+import Chart from 'chart.js/auto';
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
import Chart from "chart.js/auto";
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                      'rgba(255, 99, 132, 0.2)',
+                      'rgba(50, 250, 250, 0.2)',
+                      'rgba(23, 99, 250, 0.2)',
+                      'rgba(250, 99, 250, 0.2)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(250, 99, 250, 0.2)",
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                  data: tally,
+                  backgroundColor: [
+                      'rgba(255, 99, 132, 0.2)',
+                      'rgba(50, 250, 250, 0.2)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(50, 250, 250, 0.2)",
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                  backgroundColor: [
+                      'rgba(255, 99, 132, 0.2)',
+                      'rgba(50, 250, 250, 0.2)',
+                      'rgba(23, 99, 250, 0.2)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(23, 99, 250, 0.2)",
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+                      
+                  ],
+                  borderColor: [
+                      'rgba(255, 99, 132, 1)',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
                      "rgba(255, 99, 132, 1)",
```

eveyugyug commented on decide_panel/src/components/Graphics.js:
```diff
@@ -0,0 +1,97 @@
+const Graphics = () => {
+  const [errorConection, setErrorConection] = useState(null);
+  const [state, setState] = useState({
+    data: '{}',
```
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Full-Tortuga/decide-full-tortuga-almacenamiento/pullRequest?prid=8645779)
```suggestion
    data: "{}",
```

NoSoyDani committed:
feat: Añade formulario de selección

Se ha desarrollado un formulario que permite elegir la
gráfica correspondiente a una votación en función del id
introducido.

Se ha desarrollado la funcionalidad necesaria para evitar que
se recarge la web al enviar los datos del formulario.

dancasnar1 reviewed (approved):
Se ha comprobado la correcta visualización de la página.

dancasnar1 committed:
Merge branch 'main' into style/10-grafica_votacions

NoSoyDani committed:
fix: Modifica la URI

Se ha modificado la URI para que esta sea igual a la utilizada en otras consultas.

manmunesp12 reviewed (approved):
Se ha comprobado la correcta creación de los gráfico referentes a las diferentes votaciones

dancasnar1 committed:
Merge branch 'main' into style/10-grafica_votacions
